mipengine/controller/api/validator.py

```diff
@@ -113,12 +115,17 @@ def _validate_inputdata_pathology_and_dataset(pathology: str, datasets: List[str
         )
 
 
-def _validate_inputdata_filter(filter):
+def _validate_inputdata_filter(pathology, filter):
     """
     Validates that the filter provided have the correct format
     following: https://querybuilder.js.org/
     """
-    # TODO Add filter
+    # TODO Add filters
+    # validate_proper_filter(
+    #     common_data_elements=common_data_elements,
+    #     pathology_name=pathology,
+    #     rules=filter
+    # )
     pass
```

Comment:
Why this is not implemented?

Reply:
Really don't know... Added a bug for this: https://team-1617704806227.atlassian.net/browse/MIP-214

Some integration tests should also be added to ensure that it's always enabled.